tutorials/inputs/input_examples.rst

```diff
@@ -42,6 +42,27 @@ Examples:
             # move as long as the key/button is pressed
             position.x += speed * delta
 
+ .. code-tab:: csharp
+
+    // Input event - runs when the input happens.
+    public override void _Input(InputEvent inputEvent)
+    {
+        if (inputEvent.IsActionPressed("jump"))
+        {
+            Jump();
+        }
+    }
+
+    // Polling - runs every frame.
+    public override void _PhysicsProcess(float delta)
+    {
+        if (Input.IsActionPressed("move_right"))
+        {
+            // move as long as the key/button is pressed
```

Comment:
This one could be better too, but since this one is also present in the GDScript version, if you improved this here it would be best to also improve the GDScript comment.